src/esssans/loki/general.py

```diff
@@ -51,6 +52,16 @@ def get_monitor_data(
     return RawMonitor[RunType, MonitorType](out)
 
 
+def calibrate_monitor_position(
```

Comment:
I don't like the fact that I needed to add this. Should the `loki.get_monitor_data` just return `CalibratedMonitor` instead of `RawMonitor`?

Reply:
Yes, if we know that the raw files contain calibrated positions then we should directly return that.

Reply:
I guess it depends what we mean by "calibrated". Does it mean that the positions have been modified to be correct, or does it simply mean that the positions are correct?

If the former, then we can't really say that this would return `CalibratedMonitor` because the positions have not been modified, they are "raw".

Reply:
I changed to return `CalibratedMonitor` directly.